deployment-scripts/deploy.sh

```diff
@@ -0,0 +1,86 @@
+#! /bin/bash
+
+function usage {
+    cat << EOF
+Usage: ${0} BASE_DIR ENV GIT_HUB_DEPLOYMENT_KEY
+Deploys to the ENV environment.
+BASE_DIR is the base folder containing the configurations for all environments
+EOF
+}
+
+function error {
+    colour='\033[0;31m'
+    standard='\033[0m'
+    echo -e "${colour}ERROR: ${@}${standard}" >&2
+}
+
+
+set -eo pipefail
+# automatically export all environment variables
+set -a
```

Comment:
I think this should probably be moved down to just before the three `export` statements below.

Reply:
Sure. And if we move those statements after the exports, we can also set -euo pipefail to also catch undefined vars